test/tests/functional/pbs_init_script.py

```diff
@@ -43,23 +43,28 @@ class TestPbsInitScript(TestFunctional):
     """
     Testing PBS Pro init script
     """
+
     def test_env_vars_precede_pbs_conf_file(self):
         """
         Test PBS_START environment variables overrides values in pbs.conf file
         """
-        self.du.run_cmd(cmd=['/etc/init.d/pbs', 'stop'])
-
         conf = {'PBS_START_SERVER': '1', 'PBS_START_SCHED': '1',
-                'PBS_START_COMM': '1', 'PBS_START_MOM': '0'}
+                'PBS_START_COMM': '1', 'PBS_START_MOM': '1'}
+        self.du.set_pbs_config(confs=conf)
+
+        self.du.run_cmd(cmd=['/etc/init.d/pbs', 'stop'], sudo=True)
```

Comment:
why didn't you use PBSInitServices().stop() instead?

Reply:
I don't want PBSInitServices() to muck with pbs.conf.  The whole point of this test suite is to test if environmental variables override pbs.conf variables.  I don't think PBSInitServices() will muck with pbs.conf unless you only stop one daemon, but I don't want to take the chance.